src/Symfony/Component/HttpClient/NoPrivateNetworkHttpClient.php

```diff
@@ -178,14 +179,6 @@ public function request(string $method, string $url, array $options = []): Respo
         });
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function stream($responses, ?float $timeout = null): ResponseStreamInterface
-    {
-        return $this->client->stream($responses, $timeout);
```

Comment:
fix for the `stream()` method: the one provided by `AsyncDecoratorTrait` is fine instead (I forgot to remove this one when adding the trait)